.circleci/config.yml

```diff
@@ -117,14 +117,14 @@ workflows:
             - test_dashboard
             - build_go_images
             - build_dashboard
-      - sync_chart:
-          <<: *build_on_master
-          requires:
-            - local_e2e_tests
-            - GKE_1_17_MASTER
-            - GKE_1_17_LATEST_RELEASE
-            - GKE_1_18_MASTER
-            - GKE_1_18_LATEST_RELEASE
+      # - sync_chart:
+      #     <<: *build_on_master
+      #     requires:
+      #       - local_e2e_tests
+      #       - GKE_1_17_MASTER
+      #       - GKE_1_17_LATEST_RELEASE
+      #       - GKE_1_18_MASTER
+      #       - GKE_1_18_LATEST_RELEASE
```

Comment:
I think the git history is OK if we need to find it, rather than having it here but commented out.

Reply:
Yes, I usually agree with that. However, since I've been also working (just a bit) on the pull request approach, much of this logic is still the same. It is just the script that is changing.
Additionally, I felt that _removing_ doesn't fit the semantic of what we intend here, that is _disabling_ it temporarily until we came up with a proper approach.

However, no strong opinion here, so removing it!